Bio/SearchIO/InterproIO/__init__.py

```diff
@@ -0,0 +1,8 @@
+"""InterProScan."""
+from .interpro_xml import InterproXmlParser
+
+
```

Comment:
Would it also be useful to add more docstrings here about the format in general? For example, you can say which InterProScan output type is supported or how to access certain values. The docstrings in `BlastIO/__init__.py` and/or `HmmerIO/__init__.py` may be useful as a template.

Reply:
done